File: Python/ex049.py
#Ex049 Refaça o desafio 009, mostrando a tabuada de um número que o usuário escolher, só que agora utilizando um
# laço for.
n = int(input('Digite o número no qual você deseja saber a tabuada: '))
for c in range(1, 11):
    t = n * c
    print(f'{c} x {n} = {t}')

File: Java/Ex049.java
/*
Ex049 Refaça o desafio 009, mostrando a tabuada de um número que o usuário 
escolher, só que agora utilizando um laço for.
*/
package ex049;

import java.util.Scanner;

public class Ex049 {

    public static void main(String[] args) {
        
        Scanner t = new Scanner(System.in);
        System.out.println("Digite o número no qual você deseja saber a tabuada: ");
        int i = t.nextInt();

	for (int j = 0; j<=10; j++){
	System.out.println (i+" x "+ j + " = " + i*j);
		}
	}
}

    
    

